src/include/wiredtiger.in

```diff
@@ -1329,6 +1329,20 @@ struct __wt_session {
 	 */
 	int __F(log_printf)(WT_SESSION *session, const char *fmt, ...);
 
+#if !defined(DOXYGEN)
```

Comment:
I'm inclined to make this documented. Having a utility and code in the examples makes me think we are better to document the functionality than attempt to hide it.